docs/sanic/testing.md

```diff
@@ -59,6 +59,22 @@ the available arguments to aiohttp can be found
 [in the documentation for ClientSession](https://aiohttp.readthedocs.io/en/stable/client_reference.html#client-session).
 
 
+## Using a random port
+
+If you need to test using a random port instead of the default with 
+`SanicTestClient`, you can do so by specifying `port=None`
```

Comment:
From a general usability standpoint, wouldn't it be good to reverse the behavior? 
i.e. the `port` param defaults to `None` and picks a random port, however, you can specify a port and use for running test cases. IMHO, this would be more readable and easy to understand. 

As an additional option, we can even add an ENV variable `TEST_CLIENT_PORT` and use that to provide a much dynamic port assignment.

Reply:
I like this idea but unless other people pipe up on it I think adding this functionality and changing the default can be addressed separately

Reply:
Yeah I think changing the default functionality should come with a different PR